### PR TITLE
500 eff logo

### LINKED ISSUE #500: Add EFF logo above the fold
Let's add the EFF logo above the fold on Certbot.

Here's the solution our design team came up:

* 40 pixel thin banner above
* EFF monogram logo + text on same baseline ("a project of the Electronic Frontier Foundation"
* banner links back to EFF.org and should appear on each page.
* text: Roboto slab regular; unsure about size in rem
* colors: -- EFF logo #EC1E1E -- banner background #2E2E2E -- text: #D9D9D9

Note: Design tested these colors on webaim.org and they pass. Design team has the EFF monogram SVG.

### PR DESCRIPTION
**Pull request by lschatzkin**
closes #500

### CODE REVIEW

**Comment by bmw** [comment]
Test failures here are unrelated to your work and is due to some refactoring we've done on Certbot. We'll update things in this repo so they start passing again soon!

If we're taking too long and the test failures are annoying or slowing you down, don't hesitate to poke us.

**Comment by lschatzkin** [comment]
Okay, good to know!

**Comment by bmw** [comment]
This is fixed and merging `master` into your branches should resolve the test failures. Sorry for the trouble!

**Comment by lschatzkin** [comment]
No worries! I'll do that.

**Comment by wioux** [comment]
This pushes the header down so that it overlaps with the top of the page content:

Screenshot_2020-02-25_16-29-39

File: _includes/eff-logo.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="logo-banner">
+  <div class="logo">
```
Comment (wioux) [inline _includes/eff-logo.html]: The whitespace around here is a little off.

**Comment by lschatzkin** [comment]
> This pushes the header down so that it overlaps with the top of the page content:
> 
> Screenshot_2020-02-25_16-29-39

Weird, not on my screen... I'll look more closely.

**Comment by lschatzkin** [comment]
I've changed the banner per our conversation with Hugh yesterday.

**Comment by wioux** [comment]
Just a slight alignment issue up in the corner (logo could be more centered), otherwise lgtm.

Screenshot_2020-02-27_14-56-15

**Comment by lschatzkin** [comment]
Good catch, also I'm going to remove the tagline on mobile because that was our decision for who has your face.

**Pull request merged**